web/api/v1/api_test.go

```diff
@@ -1541,6 +1573,17 @@ func testEndpoints(t *testing.T, api *API, tr *testTargetRetriever, es storage.E
 								Health:      "unknown",
 								Type:        "alerting",
 							},
+							AlertingRule{
```

Comment:
Since we have cases of templated labels, let's test them as well to show that we match those in non-rendered form.

Reply:
Added a test, but showcased as template label in matcher not returning anything!